ui/src/app/pages/data/data-set/export-dataset-modal.tsx

```diff
@@ -208,7 +208,7 @@ export const ExportDatasetModal = ({
     return (
       <iframe
         id='export-preview-frame'
-        data-testid='export-preview-frame'
+        data-test-id='export-preview-frame'
```

Comment:
This component is one of the few components that was using RTL's default test id "data-testid". So when I switched RTL's test selector to use "data-test-id", this components associated tests started to break. Updating this attribute fixed the test:
https://docs.google.com/document/d/1LDxH33OzrWM0-ZpgNrR5WQb8yG5AmaKqd7cZlLTo1v0/edit#heading=h.yohb9h5pqgoa